tests/api/queries.test.js

```diff
@@ -42,13 +42,8 @@ describe(`[P3][Sev3][${squad}] Search API - Verify results of different queries`
     const [route] = await Promise.all([getSearchApiRoute(), execCliCmdString(setupCommands)])
     searchApiRoute = route
 
-    user = {
-      fullName: `system:serviceaccount:${ns}:${usr}`,
-      name: usr,
-      namespace: ns,
-      token: execSync(`oc serviceaccounts get-token ${usr} -n ${ns}`),
-    }
-    await sleep(20000) // Wait for the search index to get updated.
+    await sleep(20000) // Wait for the service account and search index to get updated.
+    user = await getUserContext(usr, ns)
```

Comment:
Attempt to resolve canary failures by adding a wait after creating the service accounts.